app/routes/home.tsx

```
@@ -1,18 +1,16 @@
+import Hero from "~/components/homepage/hero";
+import HowItWorks from "~/components/homepage/how-it-works";
+import WhyItWorks from "~/components/homepage/why-it-works";
+import Testimonials from "~/components/homepage/testimonials";
 import Footer from "~/components/homepage/footer";
-import Integrations from "~/components/homepage/integrations";
-import Pricing from "~/components/homepage/pricing";
-import Team from "~/components/homepage/team";
-import { api } from "../../convex/_generated/api";
 import type { Route } from "./+types/home";
 
 export function meta({}: Route.MetaArgs) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)